packages/mdx/src/index.ts

```diff
@@ -0,0 +1,152 @@
+import { jsx, IntrinsicSxElements } from '@theme-ui/core'
+import { css, get, Theme } from '@theme-ui/css'
+import {
+  ComponentType,
+  FC,
+  ReactNode,
+  DetailedHTMLProps,
+  HTMLAttributes,
+  ElementType,
+  ComponentProps,
+} from 'react'
+import styled, { StyledComponent } from '@emotion/styled'
```

Comment:
NB: more of a note for myself, but we should consider removing the `styled` dependency for v1 #832